### PR TITLE
Add small-text

### PR DESCRIPTION
**Pull request by chschroeder**
Hi,

In this PR, I added small-text to the NLP section.

Thank you for creating this repository. I already spotted a well-known name that I would not have connected to Germany before.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The projects listed here all provide frameworks to perform Natural Language Proc
...
+| <img src="res/logos/small_text.png" width="80" height="80"><br> [small-text](https://github.com/DeepLcom) | Originating from a research project at Leipzig University, [small-text](https://github.com/webis-de/small-text) offers a modular and comprehensive Python library for building experiments and applications focused on [active learning for text classification](https://aclanthology.org/2023.eacl-demo.11/). | [<img src="res/github_logo.png" alt="small-text on GitHub" width="30" height="30">](https://github.com/webis-de/small-text) [<img src="res/internet-54.png" alt="website of small-text" width="30" height="30">](https://small-text.readthedocs.io/en/latest/)
```
Comment (johko) [inline README.md]: Thanks for the addition and very cool project! Below I just corrected one little thing you missed from copy-pasting (I admit it is a bit chaotic with all the links in basically one markdown line)
```suggestion
| <img src="res/logos/small_text.png" width="80" height="80"><br> [small-text](https://github.com/webis-de/small-text) | Originating from a research project at Leipzig University, [small-text](https://github.com/webis-de/small-text) offers a modular and comprehensive Python library for building experiments and applications focused on [active learning for text classification](https://aclanthology.org/2023.eacl-demo.11/). | [<img src="res/github_logo.png" alt="small-text on GitHub" width="30" height="30">](https://github.com/webis-de/small-text) [<img src="res/internet-54.png" alt="website of small-text" width="30" height="30">](https://small-text.readthedocs.io/en/latest/)
```

**Review by johko — CHANGES_REQUESTED** [review]
Thanks for the PR, just one little nit ;)

File: README.md (hunk 1 above)
Comment (chschroeder) [reply README.md]: Oh my, I still missed this despite double-checking. Good catch, thank you! That's why copy / paste is dangerous ;) ...

**Review by johko — APPROVED** [review]
Great :slightly_smiling_face:  LGTM :+1:

**Pull request merged**